### PR TITLE
Update enhancer docs for PGSQL, Resque, Redis, Foreman

### PR DESCRIPTION
**Pull request by mfurtak**
* Introduce `foreman` to make it easier to run the Procfile configuration that we use on Heroku
* Update the README to document the steps needed to get started with running _enhancer_

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -30,6 +30,8 @@ gem 'faraday'
 # background jobs using a Redis server as the queue
 gem 'sidekiq'
 
+gem 'pg'
```
Comment (mfurtak) [inline Gemfile]: This needs to be available for all groups now that we use `foreman`, because the `worker` process will try to load `pg` as well

**Review by ohayon — APPROVED** [review]
This is awesome! Thanks @mfurtak! 🚀

File: README.md [hunk 2]
```diff
@@ -33,16 +33,67 @@ You can set the environment variable `FASTLANE_OPT_OUT_USAGE` to opt out.
...
+# Running _enhancer_ locally
+
+* Start with a `bundle install` in the project directory to make sure you have all of the dependencies
+
+## Set needed environment variables
+
+* Create a `.env` file in your _enhancer_ project root containing the desired values for:
+
+```
+ANALYTIC_INGESTER_URL=[the public URL for Analytic Ingester]
```
Comment (KrauseFx) [inline README.md]: At this point it's not clear what `Analytic Ingester` is

File: README.md [hunk 3]
```diff
@@ -33,16 +33,67 @@ You can set the environment variable `FASTLANE_OPT_OUT_USAGE` to opt out.
...
+FL_PASSWORD=[your desired password]
```
Comment (KrauseFx) [inline README.md]: It might not be clear what password, maybe phrase it as auth password to see the graphs

**Review by KrauseFx — APPROVED** [review]

**Comment by asfalcone** [comment]
Update this to be resqu instead of sidekiq

**Pull request merged**